### PR TITLE
bpo-45634: Don't combine error checks when adding `sqlite3` int constants

### PR DESCRIPTION
**Pull request by erlend-aasland**
https://bugs.python.org/issue45634

### CODE REVIEW

File: Modules/_sqlite/module.c [hunk 1]
```diff
@@ -348,49 +348,59 @@ pysqlite_error_name(int rc)
...
-    ret += PyModule_AddIntMacro(module, SQLITE_CREATE_VTABLE);
-    ret += PyModule_AddIntMacro(module, SQLITE_DROP_VTABLE);
-    ret += PyModule_AddIntMacro(module, SQLITE_FUNCTION);
-    ret += PyModule_AddIntMacro(module, SQLITE_SAVEPOINT);
+static int
+add_integer_constants(PyObject *module) {
+#define ADD_INT(ival)                                           \
+    do {                                                        \
+        if (PyModule_AddIntConstant(module, #ival, ival) < 0) { \
+            goto error;                                         \
```
Comment (gareth-rees) [inline Modules/_sqlite/module.c]: Another possibility is to `return -1;` here, avoiding the jump.

Comment (erlend-aasland) [reply Modules/_sqlite/module.c]: Yep, removing some lines are always nice. However, it's a slow path (raising exception), so the jump will probably not make a difference.

Comment (erlend-aasland) [reply Modules/_sqlite/module.c]: Applied. Thanks!

**Review by corona10 — APPROVED** [review]

**Comment by erlend-aasland** [comment]
Thanks, @corona10. I'll fix the backport.

**Pull request merged**